### ISSUE TITLE
Apply .task-overdue to .task-list-item

### ISSUE THREAD

**Issue by mrDLSable** [jamiebrynes7/obsidian-todoist-plugin#32]
I have hidden the due date with the following CSS so they appear when hovering over the task, but ideally, I would want to make the background of the task red or apply other styling to the whole task when it is overdue. Until level 4 selectors are fully implemented ([This one in particular](https://css4-selectors.com/selector/css4/relational-pseudo-class/)) applying the class to the parent is the only way to do this without JS voodoo.

```CSS
.task-list-item .task-date {
  display: none;
}

.task-list-item:hover .task-date{
  display: block;
}

.task-list-item .task-overdue{
  background: red;
}
```

**Comment by jamiebrynes7** [comment]
Hey @mrDLSable, this seems reasonable, I'll make this change in the next release 😃

**Comment by jamiebrynes7** [comment]
This has been merged and will be released in `v1.3`!
